include/zephyr/drivers/syscon/qingke.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Sorry. I'd misunderstood.
I have got the SoC has the 'System Control register' at 0x40001000.
My pointing out is wrong.

The header place to  'include/zephyr/drivers/syscon/ is correct.

I apologize for the inconvenience, but could you please restore it?

So that, Definitions related to system control registers that are scattered all over the place should be grouped together in the header. I will point this out separately.

Reply:
Updated.